Dockerfile.v3.2.x

```diff
@@ -1,14 +1,18 @@
 FROM ubuntu:18.04
 MAINTAINER Matt Bentley <mbentley@mbentley.net>
 
-# install runtime dependencies
-RUN apt-get update &&\
-  apt-get install -y libcap-dev net-tools wget &&\
-  rm -rf /var/lib/apt/lists/*
+ARG DEBIAN_FRONTEND="noninteractive"
```

Comment:
I would suggest not setting this as the default, just as a [best practice](https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles).  I would update line 10 to:

`DEBIAN_FRONTEND="noninteractive" apt-get install -y tzdata net-tools wget`

There isn't really any need to set it for the entire image when it's only needed for the `apt-get install` command and it adds back a layer.

Reply:
Ok, updated accordingly, please have another look